CodeStarterCamp_Week2/main.swift

```diff
@@ -8,5 +8,46 @@
 
 import Foundation
 
-print("Hello, World!")
+let myLottoNumbers: [Int] = [1, 2, 3, 4, 5, 6]
+var winLottoNumbers: [Int] = makeLottoNumbers()
+var myWinLottoNumbers: [Int] = checkLottoNumbers(winLottoNumbers: winLottoNumbers, myLottoNumbers: myLottoNumbers)
+printLottoNumbers(winNumber: myWinLottoNumbers)
 
+func makeLottoNumbers() -> [Int] {
+    var winNumbers: [Int] = []
+    
+    while winNumbers.count < 6 {
+        let winNumber = Int.random(in: 1...45)
+        if !winNumbers.contains(winNumber) {
+            winNumbers.append(winNumber)
+        }
+    }
+    return winNumbers
+}
+
+func checkLottoNumbers(winLottoNumbers: [Int], myLottoNumbers: [Int]) -> [Int] {
+    var myWinNumber: [Int] = []
+    
+    for number in winLottoNumbers {
+        if myLottoNumbers.contains(number) {
+            myWinNumber.append(number)
+        }
+    }
+    return myWinNumber
+}
+
+func printLottoNumbers(winNumber: [Int]) {
+    var resultString: String = ""
+    if myWinLottoNumbers.isEmpty {
+        print("아쉽지만 겹치는 번호가 없습니다.")
+    } else {
+        for result in myWinLottoNumbers {
+            if myWinLottoNumbers.first == result {
```

Comment:
이 부분은 전역변수를 바로 참조하는 것 보다,
전달받은 파라미터를 참조해도 될 것 같은데요?
```suggestion
    if winNumber.isEmpty {
        print("아쉽지만 겹치는 번호가 없습니다.")
    } else {
        for result in winNumber {
            if winNumber.first == result {
```

Reply:
헙 그렇네요... 전역변수로 선언했으니까 굳이 파라미터로 전달해줄 필요도 없었던것같기도 하네용.. 
알려주셔서 감사합니다!!!🤣